source/common/stats/stats_impl.h

```diff
@@ -23,6 +23,10 @@
 
 #include "absl/strings/string_view.h"
 
+extern "C" {
+#include <circllhist.h>
```

Comment:
#include "circlehist.h"

<> should be used for system includes (e.g. included with the OS or compiler). Also I would've thought this should be in a subdir.